tgui/packages/tgui/interfaces/Manufacturer/index.tsx

```diff
@@ -0,0 +1,321 @@
+                  </LabeledList>
+                </Section>
+                {data.manudrive.limit !== null ? (
+                  <Stack.Item>
+                    <Section
+                      title="Loaded Manudrive"
+                      buttons={
+                        <Button
+                          icon="eject"
+                          content="Eject"
+                          disabled={data.mode !== "ready"}
+                          onClick={() => act("manudrive", { "action": "eject" })}
+                        />
+                      }
+                      mb={1}
+                    >
+                      {data.manudrive.name}
+                      <Divider />
+                      <LabeledList>
+                        <LabeledList.Item
+                          label={"Fabrication Limit"}
+                        >
+                          {data.manudrive.limit === -1 ? "Unlimited" : `${data.manudrive.limit} ${pluralize("use", data.manudrive.limit)}`}
+                        </LabeledList.Item>
+                        {data.manudrive.limit !== -1 ? (
+                          <LabeledList.Item
+                            label={"Remaining Uses"}
+                          >
+                            {data.manudrive_uses_left}
+                          </LabeledList.Item>
+                        ) : null }
+                      </LabeledList>
+                    </Section>
+                  </Stack.Item>
+                ) : null}
+                {data.panel_open ? (
+                  <Stack.Item mb={1} >
+                    <CollapsibleWireMenu wirePanel={wirePanel} />
+                  </Stack.Item>
+                ) : null}
+                <CardInfo />
+              </Stack.Item>
+              <Stack.Item>
+                <Section
+                  title="Rockbox™ Containers"
+                  textAlign="center"
+                >
+                  {data.rockboxes.map((rockbox: Rockbox) => (
+                    <Section
+                      title={rockbox.area_name}
+                      key={rockbox.byondRef}
+                    >
+                      <LabeledList>
+                        {rockbox.ores.length !== 0 ? (rockbox.ores.map((ore: Ore) => (
+                          <LabeledList.Item
+                            key={ore.name}
+                            label={ore.name}
+                            textAlign="center"
+                          >
+                            <Button
+                              textAlign="center"
+                              onClick={() => act("ore_purchase", { "ore": ore.name, "storage_ref": rockbox.byondRef })}
+                              icon="add"
+                            >
+                              {ore.cost}⪽
+                            </Button>
+                          </LabeledList.Item>
+                        ))) : "No Ores Loaded."}
+                      </LabeledList>
```

Comment:
`"No Ores Loaded."` isn't a "good" child for `LabeledList`. Consider rendering it instead of the `LabeledList` in the case of nothing being loaded (i.e. move condition up a level).